packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -30,8 +30,13 @@ const getDefaultConfig = (isDistributable: boolean): ApmAgentConfig => {
     return {
       active: false,
       globalLabels: {},
+      centralConfig: false,
+      captureHeaders: false,
+      captureBody: false,
```

Comment:
1.in `elatic-apm-node/index.d.ts`:
```typescript
type CaptureBody = 'off' | 'errors' | 'transactions' | 'all';
```
2. is it possible to improve type safety of https://github.com/elastic/kibana/blob/aa9774faa08a2d85a45f5de35fbcbcee738dac5d/packages/kbn-apm-config-loader/src/types.ts#L24 ?

Reply:
@pgayvallet I believe we were needing some type changes from APM agent for this correct?

Reply:
AgentConfigOptions is not exported from `@elastic/apm-rum`, but we could declare a compatible interface, maybe

Reply:
Will handle in follow up. Should not affect 7.10 since we still need to add config-schema validation for 'productionizing' this feature.